### ISSUE TITLE
How to perform comparisons

### ISSUE THREAD

**Issue by lpil** [michaeljones/gleam-templates#17]
Hello!

I wished to render some HTML if a list was non-empty. I tried this:

```ginga
{% if items != [] %}
  ...
{% endif %}
```

But I got this error

```
Unexpected identifier: !=

error: 
   ┌─ ./src/todomvc/web/templates/home.gleamx:51:21
   │
51 │         {% if items != [] %}
   │                     ^^
```

How might I do this? Thank you

**Comment by michaeljones** [comment]
It isn't possible at the moment. Possibly not hard to extend to though I guess we're getting into more complex expressions now.

The project started as a prototype and I told myself that people could just pre-process the data into something that already works, ie. in this case pass something in that is already the boolean value of "items not being empty". Certainly not ideal but I was looking to support the minimum viable syntax initially.

Do you think there is any reasonable way to limit the complexity of the allow expressions or do we need to start parsing arbitrary gleam or do we just scoop up everything until the `%}` and put it in the generated code?

**Comment by lpil** [comment]
I think it could get quite challenging to try and parse and understand Gleam code, so consuming text until `}}` is probably a wiser move.

**Comment by michaeljones** [comment]
Should also be fixed by 0.11.0.
